docs/docs/using-cloudinary-image-service.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Using cloudinary image service for media optimization
+---
+
+`Cloudinary` is a cloud-based end-to-end media management platform for many of the world’s top brands. With extensive product offerings from an Image management solution, Dynamic Video Platform, and a dynamic digital asset management solution.
+
+As Plugins are at the heart of developing software with Gatsby and optimization is also really important in mordern app. The Cloudinary plugins aid in making sure media assets fit this need.
+
+In this guide you will take a look at the [gatsby-source-plugin](/packages/gatsby-source-cloudinary/) and [gatsby-transformer-cloudinary](/packages/gatsby-transformer-cloudinary/) which can be used to improve the experience of handling images on Gatsby sites.
+
+Here's a [Demo site for source plugin](https://gsc-sample.netlify.com) showcasing optimized images in a masonry grid, served from [Cloudinary](https://cloudinary.com)
+
+The `gatsby-source-plugin` is helpful if you already have some images hosted on `Cloudinary` and would like to leverage on the plugin system of Gatsby.
```

Comment:
```suggestion
The `gatsby-source-plugin` is helpful if you already have some images hosted on Cloudinary and would like to leverage on the plugin system of Gatsby.
```

Reply:
Can you explain what you mean? Is there another way to query this information? "like to leverage the plugin system" feels misleading. Perhaps it's more, if you'd like to access your images via Cloudinary.